vortex-python/python/vortex/type_aliases.py

```diff
@@ -12,6 +11,7 @@
 # TypeAliases do not support __doc__.
 IntoProjection: TypeAlias = Expr | list[str] | None
 IntoArrayIterator: TypeAlias = Array | ArrayIterator | pa.Table | pa.RecordBatchReader
+IntoArray: TypeAlias = Union[Array, "pa.Array[pa.Scalar[pa.DataType]]", pa.Table]
```

Comment:
you have pyright yelling at you to use `|`

Reply:
(and `|` dislikes strings)